lib/internal/modules/esm/hooks.js

```diff
@@ -445,20 +440,17 @@ class Hooks {
 ObjectSetPrototypeOf(Hooks.prototype, null);
 
 class HooksProxy {
-  /**
-   * The request & response segment of the shared memory. TextEncoder/Decoder (needed to convert
-   * requests & responses into a format supported by the comms channel) reads and writes with
-   * Uint8Array.
-   */
-  #data;
   /**
    * The lock/unlock segment of the shared memory. Atomics require this to be a Int32Array. This
-   * segment is used to tell the main to sleep when the worker is processing, and vice versa
-   * (for the worker to sleep whilst the main thread is processing).
+   * segment is used to tell the main to sleep when the worker is processing.
    * 0 -> main sleeps
-   * 1 -> worker sleeps
+   * 1 -> main wakes up
```

Comment:
O.o why?

Reply:
why what? the worker no longer sleeps with this implementation, so I tried to make the comment clearer.